tests/manage/mcg/test_write_to_bucket.py

```diff
@@ -3,79 +3,174 @@
 import boto3
 import pytest
 
-from ocs_ci.framework.pytest_customization.marks import aws_platform_required, filter_insecure_request_warning
-from ocs_ci.framework.testlib import ManageTest, tier1, acceptance
+from ocs_ci.framework.pytest_customization.marks import (
+    aws_platform_required,
+    filter_insecure_request_warning)
+from ocs_ci.framework.testlib import (
+    ManageTest, tier1, tier2, tier3, acceptance)
 from ocs_ci.ocs import constants
-from tests.helpers import craft_s3_command
+from tests.manage.mcg import helpers
 
 logger = logging.getLogger(__name__)
 
+PUBLIC_BUCKET = "1000genomes"
+LARGE_FILE_KEY = "1000G_2504_high_coverage/data/ERR3239276/NA06985.final.cram"
+
 
 @filter_insecure_request_warning
 @aws_platform_required
```

Comment:
shouldn't these cases run on VMWare too?

Reply:
I don't see any case specific for VMWare in ocs-ci, and as I search through the code only found VMWare nodes.
As the cases here not related to the underneath VM nodes, so I think it should work.
@ebenahar comments?

Reply:
Rebased as the master already removed it.